.github/CONTRIBUTING.md

```diff
@@ -39,7 +40,7 @@ Helpful resources
 ## What should I know before I get started?
 
 ### Get ZFS
-You can build zfs packages by following [these
+You can build zfs packages by following [these
```

Comment:
There are several whitespace changes like this.
 * Are those intended?
 * What do they actually change?

Reply:
Interesting.
I didn't even touch this...

Will look at it...

Reply:
Fixed